app/src/test/java/io/apicurio/registry/rules/validity/WsdlContentValidatorTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 JBoss Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apicurio.registry.rules.validity;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import io.apicurio.registry.AbstractRegistryTestBase;
+import io.apicurio.registry.content.ContentHandle;
+
+/**
+ * @author cfoskin@redhat.com
+ */
+public class WsdlContentValidatorTest extends AbstractRegistryTestBase {
+    @Test
+    public void testValidSyntax() throws Exception {
+        ContentHandle content = resourceToContentHandle("wsdl-schema-valid.wsdl");
+        WsdlContentValidator validator = new WsdlContentValidator();
+        validator.validate(ValidityLevel.SYNTAX_ONLY, content);
```

Comment:
I think we should perhaps have a separate file for this where the **syntax** is valid but the **semantics** are invalid.  Your test uses the same valid WSDL file for both the syntax and the semantic validity check.  That's fine, but we should **also** have a test where the syntax is valid (and therefore passes here) but the semantics are invalid.  In fact you could just re-use the `wsdl-schema-invalid-semantics.wsdl` test file I think.

Reply:
👍